### ISSUE TITLE
Request: Introduce optional caching to ExecuteScalar* methods

### ISSUE THREAD

**Issue by Swoorup** [mikependon/RepoDB#648]
We are using postgis to generate MVT tiles which is binary content. 

Example query:

```fsharp
let! tile = conn.ExecuteScalarAsync<byte[]>("select suburbs(@z,@x,@y)", param)
```

It would be great if we could introduce caching layer for `ExecuteScalar*` methods as well, as the functions are computationally expensive to calculate.

**Comment by mikependon** [comment]
The fix is now available at [RepoDb v1.12.5-beta4](https://www.nuget.org/packages/RepoDb/1.12.5-beta4).

**Comment by mikependon** [comment]
Closing this ticket now.
